README.rst

```diff
@@ -396,6 +396,33 @@ Usage example as a context-manager:
             signal=openwisp_signal,
         )
 
+``openwisp_utils.tests.TimeLoggingTestRunner``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``TimeLoggingTestRunner`` logs speed of all tests that are run and reports the slow
```

Comment:
should say 'time spent' instead of speed

Proposal:

This class extends the default DiscoverRunner provided by Django and logs the time spent by each test, making it easier to spot slow tests (by default tests that took more than 0.3 seconds). This threshold can be configured using the setting `OPENWISP_SLOW_TEST_THRESHOLD`